numpy/ma/core.pyi

```diff
@@ -777,7 +777,13 @@ class MaskedArray(ndarray[_ShapeT_co, _DTypeT_co]):
     copy: Any
     diagonal: Any
     flatten: Any
-    repeat: Any
+
+    def repeat(
+        self,
+        repeats: _ArrayLikeInt_co,
+        axis: SupportsIndex | None = ...,
```

Comment:
```suggestion
        axis: SupportsIndex | None = None,
```

Reply:
For bonus point, you could consider adding an overload for `axis: None` that returns a 1-d array.

Reply:
love that! i've added that for the existing `repeat` methods too